### PR TITLE
Added rulesets for Build2019

### PR DESCRIPTION
**Pull request by nelson-w**
Updated rules.json and session_map.json for Build2019

### CODE REVIEW

File: resource/build2019-prod/rules.json [hunk 1]
```diff
@@ -0,0 +1,386 @@
...
+      "SessionSetAccess": [
+        {
+          "ApplicableFrom": null,
+          "ApplicableTo": null,
+          "Identifier": "Session Catalog"
+        }
+      ]
+    },
+    {
+      "Identifiers": ["Non-Attending Manager"],
```
Comment (gavinbarron) [inline resource/build2019-prod/rules.json]: This shouldn't be in here, they are non-attending.

File: resource/build2019-prod/rules.json [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+        "Press",
+        "Press Staff",
+        "Crew"
+      ],
+      "Access": {
+        "SessionCatalog": true,
+        "MyIgnite": true,
+        "ScheduleBuilder": true,
+        "MobileApp": true,
+        "MeetingScheduler": true,
```
Comment (gavinbarron) [inline resource/build2019-prod/rules.json]: let's make this false please, we're not doing meetings on this show, so setting this correctly is good practice

File: resource/build2019-prod/rules.json [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+        "Press",
+        "Press Staff",
+        "Crew"
+      ],
+      "Reporting": {
+        "SessionCounts": false
+      },
+      "EvalAccess": [],
+      "SessionSetAccess": [
+        {
```
Comment (gavinbarron) [inline resource/build2019-prod/rules.json]: Technically we shouldn't assign this set to the logged in users. Also technically it doesn't matter

File: resource/build2019-prod/session_length_map.json [hunk 4]
```diff
@@ -0,0 +1,43 @@
+{
+    "SessionLengthMap": [{
+            "SessionTypeName": "Breakout",
```
Comment (gavinbarron) [inline resource/build2019-prod/session_length_map.json]: Can we please have each of the session types from the session set map listed in here?

File: resource/build2019-test/rules.json [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+        "Press",
+        "Press Staff",
+        "Crew"
+      ],
+      "Access": {
+        "SessionCatalog": true,
+        "MyIgnite": true,
+        "ScheduleBuilder": true,
+        "MobileApp": true,
+        "MeetingScheduler": true,
```
Comment (gavinbarron) [inline resource/build2019-test/rules.json]: MeetingScheduler: false please

File: resource/build2019-prod/rules.json (hunk 3 above)
Comment (nelson-w) [reply resource/build2019-prod/rules.json]: I'll remove it.

**Pull request merged**